src/app/xapian/searchservice.ts

```diff
@@ -662,7 +662,7 @@ export class SearchService {
       querytext += folderQuery(folderPath);
     }
 
-    if (unreadOnly) {
+    if (unreadOnly && !this.messagelistservice.ignoreUnreadInFolders.includes(folderPath)) {
```

Comment:
I'm not too fond of this level of intimacy between this method and MessageListService.

Would it be very annoying to just not set unreadOnly when we're querying for the Sent folder?

Reply:
Not particularly...

Reply:
Logic moved up a level